docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -0,0 +1,235 @@
+=========================================
+PostgreSQL specific aggregation functions
+=========================================
+
+.. module:: django.contrib.postgres.aggregates
+   :synopsis: PosgreSQL specific aggregation functions
+
+All of these functions are available
+from the ``django.contrib.postgres.aggregates``.
+
+.. warning::
+
+    All functions come without default aliases,
+    so you must explicitly provide one.
+    Usage example::
+
+        >> result = SomeModel.objects.aggregate(arr=ArrayAgg('somefield'))
+        >> print(result)
+        {'arr': [0, 1, 2]}
+
+
+General-purpose aggregation functions
+-------------------------------------
+
+ArrayAgg
+~~~~~~~~
+
+.. class:: ArrayAgg(expression, **extra)
+
+    Returns input values, including nulls, concatenated into an array.
```

Comment:
can we rephrase each of these to say things like "Returns a list of values .." and get ride of "Return type: ..."